rules/linux/lnx_system_info_discovery.yml

```diff
@@ -0,0 +1,48 @@
+action: global
+title: System Information Discovery
+id: 42df45e7-e6e9-43b5-8f26-bec5b39cc239
+status: stable
+description: Detects system information discovery commands
+author: Ömer Günal, oscd.community
+date: 2020/10/08
+references:
+    - https://github.com/redcanaryco/atomic-red-team/blob/master/atomics/T1082/T1082.md
+falsepositives:
+    - Legitimate administration activities
+level: low
+tags:
+    - attack.discovery
+    - attack.t1082
+---
+logsource:
+    product: linux
+    categories: process_creation
+detection:
+    selection:
+        ProcessName|endswith:
+          - '/uname'
+          - '/hostname'
+          - '/uptime'
+          - '/lspci'
+          - '/dmidecode'
+          - '/lscpu'
+          - '/lsmod'
+    condition: selection
+---
+logsource:
+    product: linux
+    categories: file_event
+detection:
+    selection:
+      type: 'PATH'
```

Comment:
since you are using `file_event` category, the `type: 'PATH'` is redundant here.

you need to remove this line or change the entire rule (second rule in the collection, that starts from line 32) to rely on `auditd` logsource instead of `file_event`

Reply:
Well, as you've chosen the way of relying on `auditd`, then `type: 'PATH'` is required here (: 
Could you please return it back? The rest of the rule looks perfect and ready to merge in my opinion (: 
Thank you!

Reply:
will tag you just in case... @omergunal

Reply:
Thanks!